servers/dsub/jobs/__main__.py

```diff
@@ -64,6 +64,7 @@ def split_env_flag(name):
 app = connexion.App(__name__, specification_dir='./swagger/', swagger_ui=False)
 app.app.config['PROVIDER_TYPE'] = args.provider_type
 app.app.config['REQUIRES_AUTH'] = bool(args.requires_auth)
+
```

Comment:
nit: just revert all changes to this file since you're no longer changing it in this PR.